scss/components/_control-bar.scss

```diff
@@ -164,55 +164,6 @@
   }
 }
 
-a.oo-volume-bar {
```

Comment:
Were these styles effectively moved into the VolumeControls component?

Reply:
Yes, since I moved the code into its own component I also separated the styles. Some are new though